apprise/plugins/NotifyStreamlabs.py

```diff
@@ -0,0 +1,278 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 <example@example.com>
+# All rights reserved.
+#
+# This code is licensed under the MIT License.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files(the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions :
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+# For this to work correctly you need to register an app
+# and generate an access token
+#
+#
+#  This plugin will simply work using the url of:
+#     streamlabs://access_token/
+#
+# API Documentation on Webhooks:
+#    - https://dev.streamlabs.com/
+#
+import requests
+
+from .NotifyBase import NotifyBase
+from ..common import NotifyType
+from ..utils import validate_regex
+from ..AppriseLocale import gettext_lazy as _
+
+
+class NotifyStreamlabs(NotifyBase):
+    """
+    A wrapper to Streamlabs Donation Notifications
+
+    """
+    # The default descriptive name associated with the Notification
+    service_name = 'Streamlabs'
+
+    # The services URL
+    service_url = 'https://streamlabs.com/'
+
+    # The default secure protocol
+    secure_protocol = 'strmlabs'
+
+    # A URL that takes you to the setup/help of the specific protocol
+    setup_url = 'https://github.com/caronc/apprise/wiki/Notify_streamlabs'
+
+    # Streamlabs Api endpoint
+    notify_url = 'https://streamlabs.com/api/v1.0/donations'
+
+    # Allows the user to specify the NotifyImageSize object
+    image_size = None
+
+    # The maximum allowable characters allowed in the body per message
+    body_maxlen = 255
+
+    # Define object templates
+    templates = (
+        '{schema}://{access_token}/',
+    )
+
+    # Define our template tokens
+    template_tokens = dict(NotifyBase.template_tokens, **{
+        'access_token': {
+            'name': _('Access Token'),
+            'private': True,
+            'required': True,
+            'type': 'string',
+            'regex': (r'^[A-Za-z0-9]{40}$', 'i')
```

Comment:
the `i` at the end references the regex to being case insensitive... so you don't need to `[A-Za-z0-9]`.  [a-z0-9]` is fine.  This is minor, as there is no harm having the `i` there.  You can also just drop the `i` and leave the regex you made (purely optional)